### PR TITLE
Installing odo with yum

### PR DESCRIPTION
**Pull request by yhontyk**
Bracnhes 4.7 and 4.8

### CODE REVIEW

File: modules/developer-cli-odo-installing-odo-on-linux.adoc [hunk 1]
```diff
@@ -6,6 +6,59 @@
...
+----
+
+. List the available subscriptions:
++
+[source,terminal]
+----
+# subscription-manager list --available --matches '*OpenShift Developer Tools and Services*'
+----
+
+. In the output for the previous command, find the field `Pool ID` for your {product-title} subscription and attach the subscription to the registered system:
```
Comment (Srivaralakshmi) [inline modules/developer-cli-odo-installing-odo-on-linux.adoc:34]: Suggest: In the output **of** the previous command

Comment (Srivaralakshmi) [inline modules/developer-cli-odo-installing-odo-on-linux.adoc:34]: Suggest: find the `Pool ID` field for

**Review by Srivaralakshmi — COMMENTED** [review]
@boczkowska Hi Yana, I have left couple of comments. PTAL.

Otherwise LGTM. Crisp and Neat job!! Thanks!

**Comment by Preeticp** [comment]
@boczkowska can we get an SME/QE LGTM for this, please?

**Comment by boczkowska** [comment]
>One question, from the end user's perspective, doing the 2 step procedures in Binary installation and Tarball installation might be faster and easier than the RHEL process, any particular reason this is at the top instead?

To be honest, I haven't given this too much though, I can switch the order. 
I guess it installing with yum can be perceived as the "safer" option as it is directly linked to RH verification process (as opposed to just downloading smth from the Internet with curl).

**Pull request merged**